### PR TITLE
Azure Digital Twins 0.1.0 SDK

### PR DESCRIPTION
**Pull request by c-ryan-k**
Includes support for 2020-03-01-preview API

### CODE REVIEW

**Comment by isra-fel** [comment]
Hi @c-ryan-k  please leave a link to your openapi spec PR, thanks

**Comment by c-ryan-k** [comment]
> Hi @c-ryan-k please leave a link to your openapi spec PR, thanks

@isra-fel 

https://github.com/Azure/azure-rest-api-specs/pull/9667

**Review by isra-fel — APPROVED** [review]
This PR LGTM. Let me make the checks pass and merge.

**Comment by c-ryan-k** [comment]
@isra-fel - is there anything holding this back from being merged?

Once it's merged, how soon until we can get it released?

File: sdk/digitaltwins/Microsoft.Azure.Management.DigitalTwins/src/Microsoft.Azure.Management.DigitalTwins.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.DigitalTwins</PackageId>
+		<Description>Provides management capabilities for Microsoft Azure Digital Twins.</Description>
+		<AssemblyTitle>Microsoft Azure DigitalTwins Management</AssemblyTitle>
+		<AssemblyName>Microsoft.Azure.Management.DigitalTwins</AssemblyName>
+		<Version>0.1.0-preview</Version>
```
Comment (weshaggard) [inline sdk/digitaltwins/Microsoft.Azure.Management.DigitalTwins/src/Microsoft.Azure.Management.DigitalTwins.csproj:11]: Why version 0.1.0-preview? I would suggest 1.0.0-preview.1

**Comment by isra-fel** [comment]
@c-ryan-k my bad, sorry
And Wes is right, the versioning guideline requires the first preview version to equal `1.0.0-preview.1`: see https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md

File: sdk/digitaltwins/Microsoft.Azure.Management.DigitalTwins/src/Microsoft.Azure.Management.DigitalTwins.csproj (hunk 1 above)
Comment (isra-fel) [inline sdk/digitaltwins/Microsoft.Azure.Management.DigitalTwins/src/Microsoft.Azure.Management.DigitalTwins.csproj:11]:
```suggestion
		<Version>1.0.0-preview.1</Version>
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**